src/PresentationalComponents/Filters/Filters.js

```diff
@@ -17,10 +17,11 @@ class Filters extends Component {
 
     componentDidUpdate (prevProps) {
         if (this.props.externalFilters !== prevProps.externalFilters) {
-            const filterKey = Object.keys(this.props.externalFilters)[0];
-            if (filterKey) {
-                this.props.setFilters({ [filterKey]: `${this.props.externalFilters[filterKey]}` });
-            }
+            let filters = {};
+            Object.entries(this.props.externalFilters).forEach(
```

Comment:
Looks like your just copying externalFilters into a new object called filters?

why not `let filters = {...this.props.externalFilters}'`

Reply:
woah, good call, we can simplify this quite a bit, sorry got a lil crazy there...